doc/source/models/model_abilities/audio.rst

```diff
@@ -245,7 +245,7 @@ Basic usage, launch model ``CosyVoice-300M-SFT``.
       -H 'Content-Type: application/json' \
       -d '{
         "model": "<MODEL_UID>",
-        "text": "<The text to generate audio for>",
+        "input": "<The text to generate audio for>",
```

Comment:
I copied the curl above: https://inference.readthedocs.io/en/latest/models/model_abilities/audio.html#speech

It might need to be modified as well?